site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -778,6 +778,30 @@ components:
                description: The IDs of the gateway groups that this device is a member of. Note that "via" and "memberOf" must not be set at the same time. The same applies for "viaGroups" and "memberOf" which must be set at the same time too. The reason is that Eclipse Hono does not support groups of gateway groups.
             "ext":
                $ref: '#/components/schemas/Extensions'
+            "status":
+               $ref: '#/components/schemas/Status'
+
+      Status:
+         type: object
+         additionalProperties: false
+         description: Optional and written only by the device registry implementation.
+            It can not be updated by the user.
+            Any data inserted in the status section by the user in a PUT request will be ignored.
+         properties:
+            "created":
+               type: string
+               format: date-time
+               description: The date and time when the device was created.
+                  The format must comply with RFC 3339, section 5.6. The representation must be UTC (no timezone offset).
+            "updated":
+               type: string
+               format: date-time
+               description: The date and time when the device was last updated.
+                  The format must comply with RFC 3339, section 5.6. The representation must be UTC (no timezone offset).
+            "last-user":
+               type: string
+               description: The user who made the last edit on this device.
+                  The format must comply with RFC 3339, section 5.6. The representation must be UTC (no timezone offset).
```

Comment:
I do not think that the user id will match the date/time pattern very oftenly ;-)